task/09-functions-n-closures-tasks.js

```diff
@@ -84,7 +142,18 @@ function getPolynom() {
  *   memoizer() => the same random number  (next run, returns the previous cached result)
  */
 function memoize(func) {
-    throw new Error('Not implemented');
+
+    let memory = null;
+    return function(arg){
+        if(memory===null){
```

Comment:
Must be indented after conditional statements "if", between operators "====", before brackets "function(arg) {"